conf/receiver.cfg

```diff
@@ -19,6 +26,8 @@ capath: /etc/grid-security/certificates
 check_crls: false
 
 [messaging]
+# Project to which SSM will pull messages from. Uncomment and populate for AMS sending
```

Comment:
Move to sep AMS section

Reply:
Actually, I'm just going to change the name of the option and tweak the message.